app/src/main/java/pl/futurecollars/invoicing/service/InvoiceControllerService.java

```diff
@@ -0,0 +1,45 @@
+package pl.futurecollars.invoicing.service;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import pl.futurecollars.invoicing.db.Database;
+import pl.futurecollars.invoicing.model.Invoice;
+
+@Service
+public class InvoiceControllerService {
```

Comment:
InvoiceControllerService is not very readable name - just InvoiceService, or smth similar